src/chunk.c

```diff
@@ -925,6 +925,30 @@ ts_chunk_get_data_node_name_list(const Chunk *chunk)
 	return datanodes;
 }
 
+bool
+ts_chunk_has_data_node(Chunk *chunk, const char *node_name)
```

Comment:
Nit: I guess chunk isn't modified so can be const too.

Reply:
Done